src/main/java/bio/terra/service/snapshot/SnapshotProject.java

```diff
@@ -43,4 +44,18 @@ public SnapshotProject dataProject(String dataProject) {
         this.dataProject = dataProject;
         return this;
     }
+
+    public List<DatasetProject> getSourceDatasetProjects() {
+        return sourceDatasetProjects;
+    }
+
+    public DatasetProject getFirstSourceDatasetProject() {
+        return sourceDatasetProjects.iterator().next();
+    }
```

Comment:
It kinda stinks that we're having to duplicate the logic in Snapshot.getFirstSnapshotSource(), but I think it makes sense since we're mostly working with the "SnapshotProject" model in the firestore code.

Reply:
Yeah, exactly.  I think it's mostly there to remind us that we'll need to change that when we support multiple sources